cli/tests/integration_tests.rs

```diff
@@ -53,6 +53,34 @@ fn x_deno_warning() {
   drop(g);
 }
 
+#[test]
+fn run_from_stdin() {
+  let cat = Command::new("cat")
+    .current_dir(util::root_path())
+    .arg("cli/tests/001_hello.js")
+    .stdout(std::process::Stdio::piped())
+    .spawn()
+    .unwrap();
```

Comment:
I'm surprised this works in Window - but apparently it does. 

Nevertheless, calling cat here shouldn't be necessary - you should be able to pipe the contents directly into stdin. Use `stdin(std::process::Stdio::piped())` below.

Reply:
haha, yeah I didnt think of windows.
The goal was to support `cat file | deno run -` so thats what I wanted to test.